pkg/controller/ec2/vpccidrblock/controller.go

```diff
@@ -104,8 +104,11 @@ func (e *external) Observe(ctx context.Context, mgd resource.Managed) (managed.E
 	response, err := e.client.DescribeVpcsRequest(&awsec2.DescribeVpcsInput{
 		VpcIds: []string{aws.StringValue(cr.Spec.ForProvider.VPCID)},
 	}).Send(ctx)
-	if err != nil && !ec2.IsCIDRNotFound(err) {
-		return managed.ExternalObservation{}, awsclient.Wrap(err, errDescribe)
+
+	if err != nil {
+		return managed.ExternalObservation{
+			ResourceExists: false,
+		}, awsclient.Wrap(resource.Ignore(ec2.IsCIDRNotFound, err), errDescribe)
```

Comment:
```suggestion
		}, awsclient.Wrap(resource.Ignore(ec2.IsVPCNotFound, err), errDescribe)
```

I think that's probably the only error we should take into consideration here since the call is `DescribeVpc`.